CustomerData/ActiveVariationTagging.php

```
@@ -0,0 +1,115 @@
+
+use Magento\Customer\CustomerData\SectionSourceInterface;
+use Nosto\Tagging\Helper\Data as NostoHelperData;
+use Nosto\Tagging\Helper\Customer as NostoHelperCustomer;
+use Nosto\Tagging\Helper\Scope as NostoHelperScope;
+use Nosto\Tagging\Helper\Variation as NostoHelperVariation;
+use Nosto\Tagging\Logger\Logger as NostoLogger;
+
+class ActiveVariationTagging implements SectionSourceInterface
+{
+    /**
+     * @var NostoHelperData
+     */
+    private $nostoHelperData;
+
+    /**
+     * @var NostoHelperCustomer
+     */
+    private $nostoHelperCustomer;
+
+    /**
+     * @var NostoHelperScope
+     */
+    private $nostoHelperScope;
+
+    /**
+     * @var NostoHelperVariation
+     */
+    private $nostoHelperVariation;
+
+    /**
+     * @var NostoLogger
+     */
+    private $nostoLogger;
+
+    /**
+     * ActiveVariationTagging constructor.
+     * @param NostoHelperData $nostoHelperData
+     * @param NostoHelperCustomer $nostoHelperCustomer
+     * @param NostoHelperScope $nostoHelperScope
+     * @param NostoLogger $nostoLogger
+     */
+    public function __construct(
+        NostoHelperData $nostoHelperData,
+        NostoHelperCustomer $nostoHelperCustomer,
+        NostoHelperScope $nostoHelperScope,
+        NostoHelperVariation $nostoHelperVariation,
+        NostoLogger $nostoLogger
+    ) {
+        $this->nostoHelperData = $nostoHelperData;
+        $this->nostoHelperCustomer = $nostoHelperCustomer;
+        $this->nostoHelperScope = $nostoHelperScope;
+        $this->nostoHelperVariation = $nostoHelperVariation;
+        $this->nostoLogger = $nostoLogger;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function getSectionData()
+    {
+        $data = [];
+        $store = $this->nostoHelperScope->getStore(true);
+        if ($this->nostoHelperData->isPricingVariationEnabled($store)
+            && !$this->nostoHelperVariation->isDefaultVariationCode(
+                $this->nostoHelperCustomer->getGroupCode()
```

Missing try catch

---

We'll let it blow up. We want it to be obvious that the price variations cannot be used.